frontend/src/components/Metrics/ConfigStep/DateRangePicker/index.tsx

```diff
@@ -20,7 +20,7 @@ export const DateRangePicker = () => {
     dispatch(updatePipelineToolVerifyState(false))
     dispatch(updateSourceControlVerifyState(false))
   }
-  const changeStartDate = (value: dayjs.Dayjs | null) => {
+  const changeStartDate = (value: any) => {
```

Comment:
must give it a type, not use `any`

Reply:
I had changed it because ' _Argument of type 'unknown' is not assignable to parameter of type 'Dayjs | null'._'